examples/data_pipeline/transforms.py

```diff
@@ -73,5 +81,24 @@ class ToLongTensor(nn.Module):
     def __init__(self):
         super(ToLongTensor, self).__init__()
 
-    def forward(self, tokens: List[int]) -> Tensor:
+    def forward(self, tokens: List[List[int]]) -> Tensor:
         return torch.tensor(tokens).to(torch.long)
+
+
+def iterate_batch(pipeline):
+    def func(data_batch):
+        return [pipeline(data) for data in data_batch]
+    return func
+
+
+class TextClassificationPipeline(nn.Module):
```

Comment:
Is this class being used anywhere?

Reply:
I deleted it by accident in previous PR. It's used in the `batch_torchtext` pipeline.

Reply:
I see this makes sense